src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -867,11 +867,16 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
                   context.make(substResetForOriginal.transform(context.tree))
                 }
                 typerWithLocalContext(resetContext) { typer1 =>
-                  val tree1 = typer1.typed(resetTree, mode)
-                  // Q: `typed` already calls `pluginsTyped` and `adapt`. the only difference here is that
-                  // we pass `EmptyTree` as the `original`. intended? added in 2009 (53d98e7d42) by martin.
-                  tree1 setType pluginsTyped(tree1.tpe, typer1, tree1, mode, pt)
-                  if (tree1.isEmpty) tree1 else typer1.adapt(tree1, mode, pt)
+                  typer1.silent { typer1 =>
+                    val tree1 = typer1.typed(resetTree, mode)
+                    // Q: `typed` already calls `pluginsTyped` and `adapt`. the only difference here is that
+                    // we pass `EmptyTree` as the `original`. intended? added in 2009 (53d98e7d42) by martin.
```

Comment:
The code base is rife with similar unanswered questions which must be historical. I wonder if such comments should be tickets, or in a todo queue. There is no test to automate resolution. So maybe periodically bump from unanswered questions queue to unanswerable questions queue.